x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/40_unsupported_types.yml

```diff
@@ -166,56 +162,54 @@ unsupported:
   - match: { columns.2.original_types: [completion] }
   - match: { columns.3.name: date_nanos }
   - match: { columns.3.type: date_nanos }
-  - match: { columns.4.name: date_range }
+  - match: { columns.4.name: dense_vector }
```

Comment:
Normally what we've done in this file is simply update the type to now be supported, as opposed to removing it from the file. This makes for a much simpler change. If you really want to remove it from the file, then consider adding additional tests to the 30_types.yml file to test the new type. As it currently stands, you no longer have date_range mentioned in any yaml tests, which is not so good.